cc/position.cc

```diff
@@ -51,12 +51,12 @@ std::array<inline_vector<Coord, 4>, kN* kN> kNeighborCoords = []() {
   return result;
 }();
 
-inline const inline_vector<Coord, 4>& GetNeighborCoords(Coord c) {
+}  // namespace
+
+const inline_vector<Coord, 4>& GetNeighborCoords(Coord c) {
```

Comment:
I thought these calls weren't getting inlined now? Or is the compiler able to inline the calls from position.cc?